modulefiles/build_noaacloud_intel.lua

```diff
@@ -5,7 +5,7 @@ the NOAA cloud using Intel-oneapi
 
 whatis([===[Loads libraries needed for building the UFS SRW App on NOAA cloud ]===])
 
-prepend_path("MODULEPATH", "/contrib/EPIC/spack-stack/spack-stack-1.4.1/envs/unified-dev/install/modulefiles/Core")
+prepend_path("MODULEPATH", "/contrib/EPIC/spack-stack/spack-stack-1.5.0/envs/unified-env/install/modulefiles/Core")
```

Comment:
The path should be:
prepend_path("MODULEPATH", "/contrib/spack-stack/spack-stack-1.5.0/envs/unified-env/install/modulefiles/Core")